clustering/common/src/main/java/org/jboss/as/clustering/controller/ReloadRequiredAddStepHandler.java

```diff
@@ -22,8 +22,6 @@
 
```

Comment:
Because it's still referenced by certain clustering resources.  This class is already planned to be removed altogether in https://github.com/wildfly/wildfly/pull/7840